.github/workflows/ci.yaml

```diff
@@ -557,16 +676,23 @@ jobs:
 
       - uses: actions/checkout@v4
 
-      - name: Set up JDK 8
+      - name: Set up JDK ${{ env.JAVA_VERSION }} (${{ env.JAVA_VENDOR }})
         uses: actions/setup-java@v4
         with:
-          java-version: '8'
-          distribution: 'temurin'
+          java-version: ${{ env.JAVA_VERSION }}
+          distribution: ${{ env.JAVA_VENDOR }}
+
+      - name: Set up JDK ${{ env.INT_TEST_JAVA_RUNTIME_VERSION }} (${{ env.INT_TEST_JAVA_RUNTIME_VENDOR }}) to run integration test
+        uses: actions/setup-java@v4
+        if: ${{ !(env.INT_TEST_JAVA_RUNTIME_VERSION == env.JAVA_VERSION && env.INT_TEST_JAVA_RUNTIME_VENDOR == env.JAVA_VENDOR) }}
+        with:
+          java-version: ${{ env.INT_TEST_JAVA_RUNTIME_VERSION }}
+          distribution: ${{ env.INT_TEST_JAVA_RUNTIME_VENDOR }}
 
       - name: Setup and execute Gradle 'integrationTestJdbc' task
         uses: gradle/gradle-build-action@v3
         with:
-          arguments: integrationTestJdbc -Dscalardb.jdbc.url=jdbc:oracle:thin:@//localhost:1521/FREEPDB1 -Dscalardb.jdbc.username=SYSTEM -Dscalardb.jdbc.password=Oracle
+          arguments: integrationTestJdbc -PintegrationTestJavaRuntimeVersion=${{ env.INT_TEST_JAVA_RUNTIME_VERSION }} -PintegrationTestJavaRuntimeVendor=${{ env.INT_TEST_JAVA_RUNTIME_VENDOR }} -Dscalardb.jdbc.url=jdbc:oracle:thin:@//localhost:1521/FREEPDB1 -Dscalardb.jdbc.username=SYSTEM -Dscalardb.jdbc.password=Oracle
```

Comment:
`JdkConfigurationPlugin` can accept `javaVersion` and `javaVendor`, but this workflow file doesn't seem to pass the information to the plugin. So, I guess the following confusing situation could occur later.
1. Decide to build ScalarDB with JDK 11
2. Revise this workflow to change `env.JAVA_VERSION` to 11
3. Indeed, JDK 11 is installed by `actions/setup-java`
4. But `JdkConfigurationPlugin` still uses the default JDK 8

To reduce future confusions and efforts, how about explicitly passing `env.JAVA_VERSION` and `env.JAVA_VENDOR` to the plugin in this PR?

Reply:
I agree, thank you.
Revised in https://github.com/scalar-labs/scalardb/pull/1749/commits/1ed1997919a3493d731e8c65b8ed41017252aad7